auditlog_tests/tests.py

```diff
@@ -1862,6 +1865,159 @@ def test_access_log(self):
+            self.my_post_log_data["my_error"] = error
+
+            self.assertEqual(len(pre_log_results), 2)
+
+            found_first_result = False
+            found_second_result = False
+            for pre_log_fn, pre_log_result in pre_log_results:
+                if pre_log_fn is pre_log_receiver and pre_log_result == my_ret_val:
+                    found_first_result = True
+            for pre_log_fn, pre_log_result in pre_log_results:
+                if (
+                    pre_log_fn is pre_log_receiver_extra
+                    and pre_log_result == my_other_ret_val
+                ):
+                    found_second_result = True
+
+            self.assertTrue(found_first_result)
+            self.assertTrue(found_second_result)
+
+            return my_ret_val
+
+        pre_log.connect(pre_log_receiver)
+        pre_log.connect(pre_log_receiver_extra)
+        post_log.connect(post_log_receiver)
+
+        self.obj = SimpleModel.objects.create(text="I am not difficult.")
+
+        self.assertSignals(LogEntry.Action.CREATE)
+
+    def test_custom_signals_update(self):
+        def pre_log_receiver(sender, instance, action, **_kwargs):
+            self.my_pre_log_data["is_called"] = True
+            self.my_pre_log_data["my_sender"] = sender
+            self.my_pre_log_data["my_instance"] = instance
+            self.my_pre_log_data["my_action"] = action
+
+        def post_log_receiver(sender, instance, action, error, **_kwargs):
+            self.my_post_log_data["is_called"] = True
+            self.my_post_log_data["my_sender"] = sender
+            self.my_post_log_data["my_instance"] = instance
+            self.my_post_log_data["my_action"] = action
+            self.my_post_log_data["my_error"] = error
+
+        pre_log.connect(pre_log_receiver)
+        post_log.connect(post_log_receiver)
+
+        self.obj.delete()
+
+        self.assertSignals(LogEntry.Action.DELETE)
+
+    def test_custom_signals_delete(self):
+        def pre_log_receiver(sender, instance, action, **_kwargs):
+            self.my_pre_log_data["is_called"] = True
+            self.my_pre_log_data["my_sender"] = sender
+            self.my_pre_log_data["my_instance"] = instance
+            self.my_pre_log_data["my_action"] = action
+
+        def post_log_receiver(sender, instance, action, error, **_kwargs):
+            self.my_post_log_data["is_called"] = True
+            self.my_post_log_data["my_sender"] = sender
+            self.my_post_log_data["my_instance"] = instance
+            self.my_post_log_data["my_action"] = action
+            self.my_post_log_data["my_error"] = error
+
+        pre_log.connect(pre_log_receiver)
+        post_log.connect(post_log_receiver)
+
+        self.obj.text = "Changed Text"
+        self.obj.save()
+
+        self.assertSignals(LogEntry.Action.UPDATE)
+
+    @patch("auditlog.receivers.LogEntry.objects")
+    def test_signals_errors(self, log_entry_objects_mock):
```

Comment:
it would be great to test also the `post_log` signal here

Reply:
They're all tested already.

Reply:
I mean the error in post_log. Did I miss it?

Reply:
Oh, I see what you mean. Tbh, initially, I just copied the tests as-is from the other PR. I just went over it now and updated a few things and added the test you suggested.